src/editor/CursorJumper.ts

```diff
@@ -25,6 +25,15 @@ export class CursorJumper {
             await app.vault.modify(active_file, new_content as string);
             this.set_cursor_location(positions);
         }
+
+        // enter insert mode for vim users
+        if (app.vault.getConfig("vimMode")) {
+           // @ts-ignore
```

Comment:
@chrisgrieser I'd prefer if the "obsidian" module was extended so we avoid using @ts-ignore